### PR TITLE
data: add go1.15.[56] and go1.14.1[23]

### PR DESCRIPTION
**Pull request by danp**
1.15.5 and 1.14.12: https://groups.google.com/g/golang-announce/c/NpBGTTmKzpM

1.15.6 and 1.14.13: https://groups.google.com/g/golang-announce/c/0kN8l6MprTo

### CODE REVIEW

**Review by edmorley — APPROVED** [review]

**Comment by danp** [comment]
We may want to hold off on these versions: https://github.com/golang/go/issues/42565. The buildpack does support CGO_LDFLAGS so there could be breakage.

**Comment by shivknight** [comment]
According to https://github.com/golang/go/issues/42565#issuecomment-727214122, the issue will be fixed in the next release. Does that mean there will not be a `go1.15.5` buildpack at all?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Go Buildpack Changelog
 
 ## Unreleased
+* Add go1.14.12, use for go1.14
+* Add go1.15.5, use for go1.15
+* Add go1.14.13, use for go1.14
+* Add go1.15.6, use for go1.15
```
Comment (edmorley) [inline CHANGELOG.md]:
```suggestion
* Add go1.14.12
* Add go1.15.5
* Add go1.14.13, use for go1.14
* Add go1.15.6, use for go1.15
```

**Review by edmorley — APPROVED** [review]

**Comment by danp** [comment]
Updated this for 1.15.6 and 1.14.13, new files have been synced. https://github.com/golang/go/issues/42565 is fixed in 1.15.6.

File: CHANGELOG.md (hunk 1 above)
Comment (danp) [reply CHANGELOG.md]: Thanks, fixed!

**Pull request merged**